### DISCUSSION TITLE
Underwater Agility and Thieving activity on Fossil Island - Mermaid Tears

### DISCUSSION THREAD

**Discussion by AndyBotwin1** [runelite/runelite#16078] [Ideas]
Is it possible to make a plugin to make obtaining Mermaid's tears easier during the Underwater Agility and Thieving activity on Fossil Island? 

At the moment, it is confusing and I'm struggling to understand how it all works as I'm sure many people are. 

Features could include the fastest route to the chest, what holes to take in the ground, a mermaid tears per hour estimate etc. 

Let me know your thoughts, would be great plugin for those working on collection log for the Merfolk Trident now and in the future. 

Thank you

**Comment by Randylicious** [comment]
There also currently appears to be an issue with Underwater Agility and Thieving when using the runelite client. Specifically, the arrows pointing to the chests do now show up on the map or minimap. However, when using the official Jagex client it works as intended, just without all the QOL features offered by runelite :)

**Reply by Nightfirecat** [reply]
Some plugin you are using is overwriting the arrow's location, you should check through your hub plugins and disable whichever one is interfering with it

**Reply by Truesh00ter** [reply]
For anyone wondering and finding this Github issue via a search engine, I've had the same issue:

- the arrows aren't flashing like normal intended behaviour
- the arrows flash only for about 20ms every 4 seconds

Fixed by dropping my opened clue (and picking it up again) which disabled the 'clue addon overlay' until I opened it again.
